mlrun/api/api/endpoints/utils.py

```diff
@@ -0,0 +1,14 @@
+import re
```

Comment:
Please move everything from here to `mlrun/api/api/utils.py`

Reply:
Done. Looking at the contents of that file they did not seem too relevant to the endpoints, but I'm happy to consolidate the files if that makes sense to you.